Sources/FuzzilliCli/CodeGeneratorWeights.swift

```diff
@@ -34,11 +34,11 @@ let codeGeneratorWeights = [
     "ArrayWithSpreadGenerator":                 5,
     "TemplateStringGenerator":                  1,
     "PlainFunctionGenerator":                   15,
-    "ArrowFunctionGenerator":                   3,
-    "GeneratorFunctionGenerator":               3,
-    "AsyncFunctionGenerator":                   3,
-    "AsyncArrowFunctionGenerator":              1,
-    "AsyncGeneratorFunctionGenerator":          1,
+    "ArrowFunctionGenerator":                   10,
```

Comment:
Accidental changes? :)

Reply:
Yeah, I bumped up these values to do some stress testing. I'll roll these back to the original values.